packages/create-bison-app/template/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
All other tsconfig files are using commonjs and this started creating a build error when running buildProd, it was not happy with the empty export.

Reply:
See my comment above,  you can still set this to esnext and override at the command level